File: codeforces/1360A Minimal Square.py
t = int(input())
while t > 0:
	x = input().split()
	c = min(int(x[0]), int(x[1]))
	l = max(int(x[0]), int(x[1]))
	w = c
	if l > 2 * w:
		print(l ** 2)
	else:
		print((2 * w) ** 2)
	t += -1

File: codeforces/1360A Minimal Square.cpp
#include <iostream>
using namespace std;

int main() {
	
	int t;
	cin >> t;
	while (t--) {
		int l, w, c;
		cin >> l >> w;
		c = min(l, w);
		l = max(l, w);
		w = c;
		if (l > 2 * w)
			cout << l * l << endl;
		else
			cout << 4 * w * w << endl;
	}

}